dynamiqs/utils/utils.py

```diff
@@ -220,27 +259,36 @@ def tensprod(*args: Tensor) -> Tensor:
     The returned tensor shape is:
 
     - $(n, 1)$ with $n=\prod_k n_k$ if all input tensors are kets with shape $(n_k, 1)$,
+    - $(1, n)$ with $n=\prod_k n_k$ if all input tensors are bras with shape $(1, n_k)$,
     - $(n, n)$ with $n=\prod_k n_k$ if all input tensors are density matrices or
       operators vectors with shape $(n_k, n_k)$.
 
-    Warning:
-        This function does not yet support tensors in the bra format.
-
-    Warning:
-        This function does not yet support arbitrarily batched tensors (see
-        [issue #69](https://github.com/dynamiqs/dynamiqs/issues/69)).
-
     Note:
         This function is the equivalent of `qutip.tensor()`.
 
     Args:
         *args (n_k, 1) or (n_k, n_k): Sequence of kets, density matrices or operators.
 
     Returns:
-        (n, 1) or (n, n): Tensor product of the input tensors.
+        (n, 1) or (1, n) or (n, n): Tensor product of the input tensors.
     """
```

Comment:
Need to add an indication of the batch size `(...)` in all tensor shapes in the dosctring.